glib/src/lib.rs

```diff
@@ -104,9 +98,7 @@ mod exit_code;
 pub use exit_code::ExitCode;
 
 pub mod collections;
-pub use collections::{
-    ptr_slice::IntoPtrSlice, strv::IntoStrV, List, PtrSlice, SList, Slice, StrV,
```

Comment:
You're not adding these two traits to the prelude. Any reason why?

Are there other traits (outside `glib::translate`) that are not in the prelude?

Reply:
Because they are specific to ffi usage so I re-exported them for translate instead.

There are various traits like FromValue that comes to mind. We are over exporting things in the main scope, I think we should reduce that going forward and just have people import things when they need them